alf/utils/summary_utils.py

```diff
@@ -169,6 +169,8 @@ def summarize_loss(loss_info: LossInfo):
     Args:
         loss_info (LossInfo): ``loss_info.extra`` must be a namedtuple
     """
+    if not loss_info.loss:
```

Comment:
bool(torch.tensor(0.0)) is False
and should only skip line 174

Reply:
fixed